src/App.css

```diff
@@ -1,38 +1,38 @@
 .App {
```

Comment:
Lol this CSS file has so many changes yet is not used at all in our app. It not stopping me from merging, I plan to get rid of this along with the index.css file as well.